src/keylessBackup/KeylessBackupProgress.tsx

```diff
@@ -87,12 +88,16 @@ function Restore() {
   const dispatch = useDispatch()
 
   const onPressTryAgain = () => {
-    ValoraAnalytics.track(KeylessBackupEvents.cab_restore_failed_try_again)
+    dispatch(keylessBackupBail())
```

Comment:
added these to clear the keylessBackup state so that if a user tries again we ask them for new info

Reply:
where is the `X` button (in the top left corner)  behavior defined? it should be identical to this case, right?

Reply:
oop, after looking at the image in the PR description again, I'm actually not seeing an X button as appears in [the designs](https://linear.app/valora/issue/ACT-779/wallet-not-found-screen)

Reply:
That's so weird. It looks like the designs were changed in Figma to remove the X but the preview in linear didn't update.